### PR TITLE
8.0 base_continent: port to new API + usability enhancements

### PR DESCRIPTION
**Pull request by alexis-via**
Update to latest OCA coding standards
Add README.rst

### CODE REVIEW

File: base_continent/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/partner-contact/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (elicoidal) [inline base_continent/README.rst]: @alexis-via Thanks for the contribution!
In the newest README template you do not need this line

File: base_continent/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Romain Deheele
+
```
Comment (elicoidal) [inline base_continent/README.rst:47]: you might want to add yourself in the contributor list?

**Comment by elicoidal** [comment]
:+1: 
Coverage is low but there is not much python code anyway
It would be awesome to preload the list of country/continent...

**Comment by antespi** [comment]
:+1: 
Travis unrelated error
It would be great to write the continent field in each country. It is a well known data, isn't it?

**Comment by elicoidal** [comment]
Is there any easy way to build a xml from a csv file?
I can organize the following into a csv with xml_id (http://www.worldatlas.com/cntycont.htm)

**Comment by alexis-via** [comment]
OK, I found a parsable list of countries <-> continents ; I'll do it.

**Comment by alexis-via** [comment]
FYI, I plan to use geonames: http://download.geonames.org/export/dump/countryInfo.txt

**Comment by alexis-via** [comment]
@antespi done.

**Comment by elicoidal** [comment]
:+1:

**Comment by rafaelbn** [comment]
Please @alexis-via , try to fix runbot and travis so I can test it. Thanks!

**Comment by rafaelbn** [comment]
```
2016-06-28 14:30:44     CRITICAL    server  openerp.service.server:949 preload_registries

Failed to initialize database `3149529-285-289094-all`.
Traceback (most recent call last):
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build
/3149529-285-289094/openerp/tools/cache.py", line 117, in lookup
    r = d[key]
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3149529-285-289094/openerp/tools/func.py", line 66, in wrapper
    return func(self, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3149529-285-289094/openerp/tools/lru.py", line 38, in __getitem__
    self[a[0]] = a[1]
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3149529-285-289094/openerp/tools/func.py", line 66, in wrapper
    return func(self, *args, **kwargs)
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3149529-285-289094/openerp/tools/lru.py", line 44, in __setitem__
    del self[obj]
  File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3149529-285-289094/openerp/tools/func.py", line 68, in wrapper
    lock.release()
RuntimeError: maximum recursion depth exceeded
```

**Comment by alexis-via** [comment]
It seems to me that the issue with travis has nothing to do with my code in this PR...

**Comment by alexis-via** [comment]
I rebased it... I hope it will turn green.

**Comment by rafaelbn** [comment]
Same errors as in https://github.com/OCA/partner-contact/pull/285#issuecomment-229971874

I have tested in a base database and it works :+1:

**Comment by rafaelbn** [comment]
Super @alexis-via . Travis not related

**Comment by alexis-via** [comment]
@rafaelbn  thanks for the merge !

**Pull request merged**